beacon/src/beacon.rs

```diff
@@ -92,6 +92,7 @@ impl TryFrom<Beacon> for poc_lora::LoraBeaconReportReqV1 {
             frequency: v.frequency,
             channel: 0,
             datarate: v.datarate as i32,
+            tmst: 0,
```

Comment:
this felt like as good a default value as any since we override the values immediately after the report is created but before sending to the ingestor

Reply:
i think you could say the same for the `tx_power` being set to 27 here as well (i believe that's supposed to be the max supported?); do we want to leave that as-is or use the same mechanism to default it there too with the understanding that we intend to override in any circumstance?

Reply:
True.. leave as 0 then. works fine